application/repo-manager.jsx

```diff
@@ -137,13 +156,13 @@ export default class RepoNew extends React.Component {
                                 { st.result.actions.map((act) => { return <li key={act}>{act}</li>; }) }
                             </ul>
                             <p>
-                                You can view your { st.isNew ? "newly minted" : "imported" } repository over 
+                                You can view your { st.mode === "new" ? "newly minted" : "" } repository over 
                                 there: <a href={"https://github.com/" + st.result.repo} target="_blank">{st.result.repo}</a>
                             </p>
                           </div>;
             }
         }
-        if (st.isNew) {
+        if (st.mode === "new") {
```

Comment:
(It'd be just two `case`s and one `default`. I think @dontcallmedom is avoiding `switch`; it's discouraged by many style guides. That's what I do, anyway :)